processor/batchprocessor/metadata.yaml

```diff
@@ -46,3 +46,36 @@ telemetry:
       sum:
         value_type: int
         async: true
+
+config:
+  type: object
+  additionalProperties: false
+  description: "Configuration parameters for the batch processor."
+  properties:
+    timeout:
+      type: string
+      format: duration
+      # TODO: The default duration should be "P200MS"
+      default: "200ms"
+      description: 
+    send_batch_size:
+      type: integer
+      minimum: 0
+      default: 8192
```

Comment:
Setting a default value is important, because otherwise g--jsonschema will autogenerate an `*int`. Similarly, for `string` properties which don't have a default value it'd generate a `*string`. If we do set a default value, we will get an `int` and a `string`, with no pointers.